docs/docs/cmd/spo/folder/folder-roleinheritance-reset.md

```diff
@@ -32,5 +32,5 @@ m365 spo folder roleinheritance reset --webUrl "https://contoso.sharepoint.com/s
 Reset inheritance of folder with site-relative url _Shared Documents/TestFolder_ located in site _https://contoso.sharepoint.com/sites/project-x_. It will **not** prompt for confirmation before resetting.
```

Comment:
Hi @TheZal, could you update this line as well, mentioning the server-relative URL?

Reply:
Did that, it's ok?

Reply:
I'll look at it. Normally, if you're ready resolving PR comments, you should push the 'Ready for review button'. But that's okay for now, I can also do it for you.

Reply:
Sorry I'm new at contributing on github!